content/blog/2020-03-02-react-v16.13.0.md

```diff
@@ -0,0 +1,154 @@
+    );
+  }
+}
+```
+
+You most likely don't have code like this but if you do, we recommend that you convert it to [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) instead:
+
+```jsx
+class ClassWithRenderProp extends React.Component {
+  foo = React.createRef();
+  componentDidMount() {
+    something(this.foo.current);
+  }
+  render() {
+    return this.props.children(this.foo);
+  }
+}
+
+class ClassParent extends React.Component {
+  render() {
+    return (
+      <ClassWithRenderProp>{foo => <View ref={foo} />}</ClassWithRenderProp>
+    );
+  }
+}
+```
+
+Fixing these edge cases will be a prerequisite to do before being able to apply an automatic codemod.
+
+> To get this warning you need to have the [babel-plugin-transform-react-jsx-self](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx-self) installed in your Babel plugins. 
+> 
+> If you use the “react” preset you should already have this installed by default!
+
+### Renaming `ReactDOM.unstable_createPortal` to `ReactDOM.createPortal`
+
+Portals are now considered first class supported APIs, and as such we're removing the `unstable_` prefix for the method that creates portals. Replace all calls to `ReactDOM.unstable_createPortal` with `ReactDOM.createPortal`.
+
+### Deprecating `React.createFactory`
+
+[`React.createFactory`](https://reactjs.org/docs/react-api.html#createfactory) is a legacy helper for creating React elements. This release adds a deprecation warning to the method, and will be removed in a future major version. Replace usages of `React.createFactory` with regular JSX.
+
+### Other warnings
+
+- When dynamically changing `style` on a dom element, it's sometimes possible that the applied style clashes with a previously set style, resulting in a badly styled element. React now detects this pattern and logs a warning. The workaround is to pass the same object 'shape' to `style` everytime. 
+
+- The content of a `<textarea>` element should be passed as `value` or `defaultValue` props. An older pattern was to pass `children` to the `<textarea>` element. This release adds a warning for this usage, and support for this will be removed in the next major version.
+
+### Notable bugfixes
+
+This release contains a few other notable improvements:
+
+- In Strict Development Mode, React calls lifecycle methods twice to flush out any possible unwanted side effects. This release adds that behaviour to `shouldComponentUpdate`. This shouldn't affect most code, unless you have side effects in `shouldComponentUpdate`. To fix this, move the code with side effects into `componentDidUpdate`.
+
+- In Strict Development Mode, the warnings for usage of the legacy context API didn't include the component that actually triggered the warning. This release adds the first component stack to the warning.
+
+- Further, this release adds a component stack to a number of warnings that previously didn't include one.
```

Comment:
Maybe we want to call out stacks for hydration warnings? People have been asking for them for a long time. I would even include a separate section with a screenshot.